setup.php

```diff
@@ -28,18 +28,21 @@
  * -------------------------------------------------------------------------
  */
 
+/** @var array $CFG_GLPI */
+global $CFG_GLPI;
+
 define('PLUGIN_FIELDS_VERSION', '1.21.22');
```

Comment:
```suggestion
define('PLUGIN_FIELDS_VERSION', '1.22.0-beta1');
```

Reply:
You need to move the `css`, `js` folder into a newly created `public` directory, and update the code to reflect this new path.